<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDomainDialog</class>
 <widget class="QDialog" name="AddDomainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="DomaintextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>261</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="OkPushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelPushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
